src/Symfony/Bundle/FrameworkBundle/Resources/config/validator.xml

```diff
@@ -37,6 +37,7 @@
 
         <service id="validator.mapping.cache.apc" class="%validator.mapping.cache.apc.class%" public="false">
             <argument>%validator.mapping.cache.prefix%</argument>
+            <deprecated>The "%service_id%" service is deprecated since Symfony 2.8 and will be removed in 3.0.</deprecated>
```

Comment:
The [`ApcCache` class](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Validator/Mapping/Cache/ApcCache.php#L14) is actually deprecated since 2.5, but the deprecation tag was added on the service in 2.8 as a feature (https://github.com/symfony/symfony/pull/16011). So I guess it's still correct to fix this in the 2.8 branch.

Anyway, I'll update the `depracated` tag to mention the right version. Thanks.